internal/versions/versions.go

```diff
@@ -57,6 +57,10 @@ const (
 	NodeMaintenanceOperatorCatalogImage = "ghcr.io/edgelesssys/constellation/node-maintenance-operator-catalog"
 	NodeMaintenanceOperatorVersion      = "v0.13.1-alpha1"
 
+	// ConstellationQEMUImageURL is the artifact URL for QEMU qcow2 images.
+	// TODO: Replace with actual artifact URL once we have one (S3, Google Bucket or similar).
+	ConstellationQEMUImageURL = "http://localhost/files/constellation.qcow2"
```

Comment:
Do we want to change this before the merge? Since we already want to prepare the release this week - next week.

Reply:
We don't have any images yet, so we would just change it form one placeholder to another until the next release.
Although if you want, we could decide now under which URL we plan to upload the image already set it here.